.travis.yml

```diff
@@ -28,7 +24,8 @@ env:
         - secure: "vJ1TnFhU9muqLwkiBd13lFqdPJIrwKJ5OJjR3/jQROWQsRO4NPd/I7OfdDKk\noJGYbgjGMw7bmBYI0xjhYfZAnrClwIBvn1Qx5x5UDGxTHC8ZP6xs49AGa37I\n7BJlWC7+nRwPmmLhoLsbut/O7Kkn8IkdJIeChHbdn8RHQIJTJ4Y="
 sudo: false
 before_script:
-  - composer require --ignore-platform-reqs 'atoum/reports-extension:^2.0.1'
+  - composer remove 'atoum/reports-extension'
```

Comment:
Why do you need to remove this ?

Reply:
because of cache of travis (could be removed after the merge)